spec/plsql/package_spec.rb

```diff
@@ -35,6 +35,24 @@
     expect(PLSQL::Package.find(plsql, :test_package)).not_to be_nil
   end
 
+  context "with current_schema set" do
+    before(:all) do
+      @old_connection = plsql.connection
+      plsql.connection = get_connection(1)
+      plsql.execute "ALTER SESSION set current_schema=#{DATABASE_USERS_AND_PASSWORDS[0][0]}"
+    end
+    it "should find existing package with current_schema set" do
```

Comment:
I believe it's good to also have the scenario when the current_schema is not changed and it fails to find the package.